raiden/raiden_service.py

```diff
@@ -225,7 +231,10 @@ def matches(self, token_network_address: TokenNetworkAddress, amount: PaymentAmo
         return token_network_address == self.token_network_address and amount == self.amount
 
 
-class RaidenService(Runnable):
+T_Handler = TypeVar("T_Handler", bound=EventHandler)
```

Comment:
The prefix `T_` is weird here.

Usually `T_ADDRESS` `T_BALANCE` and so on are the runtime types corresponding to static `Address` and `Balance`.